Foxtrot/Driver/Rewriting/PostRewriteChecker.cs

```diff
@@ -0,0 +1,120 @@
+﻿using System;
+using System.CodeDom.Compiler;
+using System.Collections.Generic;
+using System.Compiler;
+using System.Linq;
+using System.Text;
+
+namespace Microsoft.Contracts.Foxtrot
+{
+    public sealed class PostRewriteChecker : Inspector
```

Comment:
I think, additional comment would be helpful that will explain the whole purpose of the class and what problem it solves...